python/GafferUITest/CompoundEditorTest.py

```diff
@@ -114,5 +116,23 @@ def testReprLifetime( self ) :
 
 		self.assertEqual( wc(), None )
 
+	def testBoundCompatability( self ) :
```

Comment:
Compatability -> Compat**i**bility

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3167/commits/4361ca224e657be1db373d6d0fe91d18a252c235